logger/drain/drain.go

```diff
@@ -17,12 +20,30 @@ func SendToDrain(m string, drain string) error {
 	switch u.Scheme {
 	case "syslog":
 		sendToSyslogDrain(m, uri)
+	case "https":
+		sendToHttpsDrain(m, uri)
 	default:
 		log.Println(u.Scheme + " drain type is not implemented.")
 	}
 	return nil
 }
 
+func sendToHttpsDrain(m string, drain string) error {
+	buf := strings.NewReader(m)
+
+	tr := &http.Transport{
+  	TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
We skip certificate verification here since we do not have root certs installed in alpine linux, and can therefore not verify certs issued by public root CAs. I think this is acceptable because we are in control of the endpoint, and since this would presumably be run in a non-hostile network, the chances of a MITM attack are low. The alternative here is to install root certs on the base image, or let the user specify a pinned cert. Specifying a cert to pin would be preferable from a security standpoint, although since the cert of a log aggregator may be out of the user's control, and could be changed at any time, it is somewhat brittle.